go.mod

```diff
@@ -23,13 +23,15 @@ require (
 	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200321030439-57b580e57e88
 	github.com/operator-framework/operator-marketplace v0.0.0-20190919183128-4ef67b2f50e9
-	github.com/spf13/cobra v1.0.0
+	github.com/sirupsen/logrus v1.7.0 // indirect
+	github.com/spf13/cobra v1.1.1
```

Comment:
Is there reason why we upgraded our cobra version?
The unit test passes when we downgrade to our older version `v1.0.0`
Can we change it back and if we want to update the cobra version can be done in a separate issue with the fix of unit test?
It's the output differs in a failing command with this version of cobra